jetty-ee10/jetty-ee10-tests/jetty-ee10-test-integration/src/test/java/org/eclipse/jetty/ee10/test/DeploymentErrorTest.java

```diff
@@ -101,8 +98,9 @@ public Path startServer(Consumer<Path> docrootSetupConsumer) throws Exception
 
         System.setProperty("test.docroots", docroots.toAbsolutePath().toString());
         DeploymentScanner deploymentScanner = new DeploymentScanner(server, deployer);
+        assertNotNull(ServletContextHandler.ENVIRONMENT, "Expected environment does not exist");
         DeploymentScanner.EnvironmentConfig envConfig = deploymentScanner.configureEnvironment("ee10");
-        envConfig.setContextHandlerClass("org.eclipse.jetty.ee10.webapp.WebAppContext");
+        envConfig.setDefaultContextHandlerClass("org.eclipse.jetty.ee10.webapp.WebAppContext");
```

Comment:
Rename to `setDefaultContextHandlerClassName(String)` in case we want to have later a version that takes a `Class<?>` parameter.